lib/AST/ModuleDependencies.cpp

```diff
@@ -707,7 +711,7 @@ bool SwiftDependencyScanningService::setupCachingDependencyScanningService(
 
     // CASOption mismatch, return error.
     Instance.getDiags().diagnose(
-        SourceLoc(), diag::error_cas,
+        SourceLoc(), diag::error_cas_setup,
```

Comment:
> error: CAS setup error: conflicting CAS options used in scanning service

How can this error occur? Is this something the user misconfigured or more of a scanner/build-system bug?

Reply:
Yes, this a mis-configuration.

Reply:
Might be nice to have a specific error in that case and not talk about "CAS setup" but just say there are conflicting "-cas-*" options?